staging/src/k8s.io/cloud-provider/controllers/node/node_controller_test.go

```diff
@@ -436,14 +436,14 @@ func Test_syncNode(t *testing.T) {
 					CreationTimestamp: metav1.Date(2012, 1, 1, 0, 0, 0, 0, time.UTC),
 				},
 				Spec: v1.NodeSpec{
-					ProviderID: "fake://12345",
```

Comment:
This change here seems not related, can we undo this just to keep the git history clean?

Reply:
(move line 446 back here)

Reply:
ah sure, good catch!